consultation_analyser/consultations/management/commands/import_consultation_data.py

```diff
@@ -0,0 +1,111 @@
+import json
+import os
+
+import boto3
+import pandas as pd
+from django.core.management.base import BaseCommand
+
+from consultation_analyser.consultations.models import (
+    Answer,
+    Consultation,
+    ExecutionRun,
+    Framework,
+    Question,
+    QuestionPart,
+    Respondent,
+    Theme,
+    ThemeMapping,
+)
+
+
+class Command(BaseCommand):
+    help = "Import data from an S3 bucket"
+
+    def add_arguments(self, parser):
+        parser.add_argument("key_prefix", nargs="+", type=str)
+
+    def handle(self, *args, **options):
+        self.stdout.write("Importing data")
+
+        key_prefix = options["key_prefix"][0]
+
+        question_key = f"{key_prefix}/expanded_question.txt"
```

Comment:
I think for the "real thing" (i.e. some future polished version) we want to use the non-expanded question, but I think that we probably have to do more thinking on how we get all the data in anyway.

For now - using the expanded question looks good!

Reply:
yes this was a case of using the data we could get in easily